en/micro-integrator/docs/setup/security/securing_management_api.md

```diff
@@ -4,17 +4,29 @@ The Management API of WSO2 Micro Integrator is an internal REST API, which was i
 the **admin services** that were available in WSO2 EI 6.x.x.
 
 The [Micro Integrator CLI](../../../administer-and-observe/using-the-command-line-interface) and the [Micro Integrator dashboard](../../../administer-and-observe/working-with-monitoring-dashboard) communicates with this service to
-obtain administrative information of the server instance. If required, you can [directly access the management API](../../../administer-and-observe/working-with-management-api) without using the dashboard or CLI.
+obtain administrative information of the server instance and to perform various administration tasks. If required, you can [directly access the management API](../../../administer-and-observe/working-with-management-api) without using the dashboard or CLI.
 
-## JWT-based user authentication
+See the topics given below information on securing the management API.
 
-JWT-based user authentication is enabled in the Micro Integrator by default.
+## Authentication (JWT)
 
-If you want to update the defualt settings, open the `deployment.toml` file (stored in the `MI_HOME/conf/` directory) and add the following:
+!!! Info
+    User authentication in the management API uses the file-based user store by default. If required, you can configure an [LDAP](../../../setup/user_stores/setting_up_a_userstore/#configuring-an-ldap-user-store) or [RDBMS](../../../setup/user_stores/setting_up_a_userstore/#configuring-an-rdbms-user-store) user store.
+
+JWT-based user authentication is enabled for the management API by default. This ensures that users that log in to the management API or log out will be authenticated.
+
+The following resources of the API handles login and logout:
+
+-       `/login`: This resource is used to obtain a JWT token based for the provided user name and password and it is protected by basic auth.
+-       `/logout`: This resource is used to revoke the JWT token.
+
+When you [access the management API directly](../../../administer-and-observe/working-with-management-api), you must first acquire a JWT token with your valid username and password. To log out of the management API, this token must be revoked. See [securely invoking the management API](../../../administer-and-observe/working-with-management-api/#securely-invoking-the-api) for more information.
+
+When you use the [Micro Integrator Dashboard](../../../administer-and-observe/working-with-monitoring-dashboard) or the [Micro Integrator CLI](../../../administer-and-observe/using-the-command-line-interface), JWT token-based authentication is handled internally.
 
 ### Disable user authentication
 
-If security is **not required**, you can simply disable the handler for the Micro Integrator:
+If security is **not required**, you can simply disable the handler for the Micro Integrator. Open the `deployment.toml` file (stored in the `MI_HOME/conf/` directory) and add the following configuration:
```

Comment:
shall we add a note here that , dashboard won't work  / can't be used if we disable this.